mbus/mbus-protocol.c

```diff
@@ -3901,7 +3901,7 @@ mbus_data_variable_header_xml(mbus_data_variable_header *header)
 /// Generate XML for a single variable-length data record
 //------------------------------------------------------------------------------
 char *
-mbus_data_variable_record_xml(mbus_data_record *record, int record_cnt, int frame_cnt, mbus_data_variable_header *header)
+mbus_data_variable_record_xml(mbus_data_record *record, int record_cnt, int frame_cnt)
```

Comment:
used variables in function

Reply:
I understand this is unused, but i would like to keep it as long these functions aren't fully implemented yet.